src/main/java/com/crowdin/client/bundles/model/FileCollectionResourceResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.crowdin.client.bundles.model;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.Date;
+import java.util.List;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class FileCollectionResourceResponse extends FileInfoCollectionResourceResponse{
```

Comment:
Let's use the `com.crowdin.client.sourcefiles.model.File` class instead of creating this duplicate